mkdocs-project-dir/docs/Account_Services.md

```diff
@@ -1,68 +1,137 @@
+<!---  @contributor(s) e.kotsialos@ucl.ac.uk                             --->
+<!---  Copyright (c) 2020, UCL:ISD:RITS, RCAS                            --->
+<!---                                                                    --->
+<!--- ================================================================== --->
+
 # Account Services
 
 ## Cluster access application
 
-There is a single online form for applying to the majority of our clusters. Use of these services is subject to a common set of [terms and conditions](Terms_and_Conditions.md). All granted applications give you a Myriad account while access to other systems depends on what types of work you tell us you will be doing. 
+There is a single online form for applying to the majority of our clusters.
+Use of these services is subject to a common set of
+[terms and conditions](Terms_and_Conditions.md).
+All granted applications give you a Myriad account while access to other systems
+depends on what types of work you tell us you will be doing.
 
 ### Account sponsors
 
-If you are a student or postdoctoral researcher, your application must be approved by a permanent member of staff (normally your supervisor or PI). This will be automatically determined when you begin your application, and when you submit it, an email will be sent to this person asking them to approve the application before the account can be created.
+If you are a Student or Postdoctoral Researcher, your application must be
+approved by a permanent member of staff (normally your Supervisor, or your PM,
+or your PI). This will be automatically determined when you begin your
+application, and when you submit it, an email will be sent to this person asking
+them to approve the application before the account can be created.
 
-Please note that the form requires your sponsor's UCL username and not their UPI.
+Please note that the form requires your sponsor's UCL username and _not_ their UPI.
 
-Permanent members of staff do not need a sponsor and their accounts will be automatically approved. 
+Permanent members of staff do not need a sponsor and their accounts will be
+automatically approved.
 
 ### Apply for an account
 
-The online application form can be found here: 
+The online application form can be found here:
+
+* [User account application form](https://signup.rc.ucl.ac.uk/computing/requests/new)
+  (UCL login required via Shibboleth)
+
+<!--- TODO: (them): is this still valid? --->
 
-* [User account application form](https://signup.rc.ucl.ac.uk/computing/requests/new) (UCL login required via Shibboleth) 
+You will need a UCL user name and password. These are the same credentials used
+to access core services like Portico and HR systems.
 
-You will need a UCL user name and password. These are the same credentials used to access core services like Portico and HR systems. 
+<!--- TODO: (them): perhaps links for Portico and "HR systems"? --->
```

Comment:
Don't think we need links to these - more things to break when the pages move, and they aren't relevant to the contents of our docs, just used as examples.